api/util/registerUser.js

```diff
@@ -0,0 +1,130 @@
+const User = require('../models/User');
+const config = require('../config/config');
+
+/**
+ * Register a new user.
+ * @param {Object} newUser - The user that is to be registered
+ * @param {(string|undefined)} - The information of the User
+ *                               that is to be added
+ */
+
+
+async function registerUser(req){
+  let res = {
+    userSaved: true,
+    message: '',
+    status: 'OK'
+  };
+  if (req.email && req.password) {
+    const newUser = new User({
+      password: req.password,
+      firstName: req.firstName,
+      middleInitial: req.middleInitial || '',
+      lastName: req.lastName,
+      email: req.email.toLowerCase(),
+      major: req.major || ''
```

Comment:
I think that u can just leave it as password, firstName, middleInitial, etc. The req. '' isn't required.

Reply:
yes! can you change the parameters of the function?

Reply:
Because i'm planning on using this register function for the terminal app to create a user as well, i'm passing in a general `req` object to the function and not individual parameters like `firstName`, `lastName`, `email`, etc.. 
So, it doesn't feel right to add more parameters because eventually I need to also manipulate the User's access level and their email verified value from the terminal application. If I do it this way I would need to add another two parameters to the function, and add two dummy values to pass in from the routes end. I can still change it and do it differently as to how I have it but I don't see the need to atm, do you still want me to do it?

Reply:
![image](https://user-images.githubusercontent.com/36381158/79033653-d349da00-7b64-11ea-86fb-dc71ee05f4af.png)

Reply:
![Alt Text](https://media.giphy.com/media/3DnDRfZe2ubQc/giphy.gif)